app/test/features/general/url_validation_test.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:acter/common/utils/utils.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
```

Comment:
@gnunicorn 
Added UNIT TEST for Regex validation. All Test are passed with given regex!